src/includes/getting-started-config/unity.mdx

```diff
@@ -1,17 +1,12 @@
-Currently, you can only configure Sentry using the Unity tools window.
-
-The minimum required configuration is to set your [DSN](/product/sentry-basics/dsn-explainer/) in the window. The DSN is available either in your Sentry project settings or, if you're logged in, by copying:
+The minimum configuration required is the [DSN](/product/sentry-basics/dsn-explainer/) to your project.
+You can find the DSN in your project settings on [sentry.io](https://sentry.io/). If you are logged in you can also select your project and copy its DSN directly from here:
```

Comment:
No need to explain the DSN on this page (as we have a link to an explanation of it). 
```suggestion
The minimum configuration required is the [DSN](/product/sentry-basics/dsn-explainer/) for your project.
```

Reply:
I removed the DSN explainer but moved the "get it from here"-part into a note below because if you're not logged in it just looks like a fancy code-snippet.